CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ You can find its changes [documented below](#060---2020-06-01).
 - Ensure that `update` is called after all commands. ([#1062] by [@jneem])
 - X11: Support idle callbacks. ([#1072] by [@jneem])
 - GTK: Don't interrupt `KeyEvent.repeat` when releasing another key. ([#1081] by [@raphlinus])
+- X11: Set some more common window properties. ([#1097] by [@psychon])
```

Comment:
The link for `#1097`  has to be added manually.

Reply:
Whoops, I missed that part. And it even says so in `CONTRIBUTING.md`...